ckan/controllers/user.py

```diff
@@ -156,6 +158,28 @@ def me(self, locale=None):
         user_ref = c.userobj.get_reference_preferred_for_uri()
         h.redirect_to(locale=locale, controller='user', action='dashboard')
 
+    def check_permissions(self):
+        organisations = []
+
+        user_organisations = h.organizations_available(permission='read')
+        for uo in user_organisations:
+            organisations.append(uo['name'])
+
+        # Note: the Ausmicro org name on prod is: 'australian-microbiome'
+        data_portion = {
+            'timestamp': time.time(),
+            'organisations': organisations
+        }
+
```

Comment:
could you add in the user's registered email as well?

Reply:
Fixed.